modin/pandas/test/dataframe/test_udf.py

```diff
@@ -399,4 +406,7 @@ def test_transform(data, func):
     eval_general(
         *create_test_dfs(data),
         lambda df: df.transform(func),
+        # older pandas gives different exception type compared to Modin and current pandas,
```

Comment:
I wonder if the user expect the exception from Modin, which was in older pandas versions, should we raise that exception?

Reply:
this would complicate our base without much profit; I'd rather not try to support every nook and cranny of older pandas and wait until users explicitly request that support

Reply:
makes sense